### PR TITLE
Widespace integration

### PR DESCRIPTION
**Pull request by jonasmattsson1**

### CODE REVIEW

File: ads/widespace.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function widespace(global, data) {
+  checkData(data, ['sid']);
+  validateDataExists(data, ['sid']);
+
+  const url = 'https://engine.widespace.com/map/engine/dynamic?isamp=1&sid=' + encodeURIComponent(data.sid);
+
+  writeScript(global, url);
```
Comment (cramforce) [inline ads/widespace.js]: Can you use `loadScript` instead?

Comment (jonasmattsson1) [inline ads/widespace.js]: We uses document.write so I think have to use writeScript?

Comment (cramforce) [inline ads/widespace.js]: You do, but you shouldn't. It isn't a problem in AMP, but very bad for the web.

Comment (jonasmattsson1) [inline ads/widespace.js]: Yes I know, we have two kinds of implementations. 
This one that we use in iframe that are more simpel but use document.write. And one that is async but is larger and create its own sandboxed iframe and expose an api. 

On AMP we think the the lighter implementation with document.write will work better as we are in your iframe anyway and we lose the benefits with our api etc. Also to have your iframe over our sandboxed iframe feels little wrong. So if we can use this implementation I think it is better.

Comment (cramforce) [inline ads/widespace.js]: SGTM

**Comment by cramforce** [comment]
This is LGTM.

I restarted the build and will merge if it succeeds. It failed due to a Travis outage.

**Comment by cramforce** [comment]
Please ping back, if I forget to merge when the build is through :)

**Comment by cramforce** [comment]
There is a type error. Need to declare the `sid` property in `ads.extern.js`

File: ads/widespace.js (hunk 1 above)
Comment (jonasmattsson1) [inline ads/widespace.js]: Thank you, now the test´s passed :)

**Pull request merged**